src/include/network/lan/address.rb

```diff
@@ -1473,7 +1473,15 @@ def AddressDialog
           #		 "No IP address" case, then default gw must stay (#460262)
           # and also: don't delete default GW for usb/pcmcia devices (#307102)
           if LanItems.isCurrentDHCP && !LanItems.isCurrentHotplug
-            Routing.RemoveDefaultGw
+            remove_gw = Routing.default_gw? && Popup.YesNo(
+              _(
+                "A static default gateway is defined.\n" \
```

Comment:
Just not sure about the wording.

Reply:
```
            _(
                "A static default route is defined.\n" \
                "It is suggested to remove the static default route \n" \
                "when it can be set also via DHCP.\n" \
                "Do you want to remove it now?"
             )
```